Directory.Build.props

```diff
@@ -3,4 +3,30 @@
   <PropertyGroup>
     <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)\kubernetes-client.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
+  
+  <PropertyGroup>
+    <Authors>The Kubernetes Project Authors</Authors>
+    <Copyright>2017 The Kubernetes Project Authors</Copyright>
+    <Description>Client library for the Kubernetes open source container orchestrator.</Description>
+
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <PackageProjectUrl>https://github.com/kubernetes-client/csharp</PackageProjectUrl>
+    <PackageIconUrl>https://raw.githubusercontent.com/kubernetes/kubernetes/master/logo/logo.png</PackageIconUrl>

```

Comment:
Maybe we could use the package icon file since the URL is deprecated

https://docs.microsoft.com/en-us/nuget/reference/nuspec#iconurl
https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#packing-an-icon-image-file

Reply:
thanks, added #797 and lets fix it in a separate PR